modules/gitops-release-notes-1-18-2.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-18.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-18-2_{context}"]
+= Release notes for {gitops-title} 1.18.2
+
+{gitops-title} 1.18.2 is available on {OCP} 4.14, 4.16, 4.17, 4.18, and 4.19.
+
+[id="errata-updates-1-18-2_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-18-2-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.18.2 security update advisory
+
+Issued: 2025-12-10
+
+The list of security fixes included in this release is available in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-18-2_{context}"]
+== New features
+
+* With this update, the Redis version included with {gitops-title} is upgraded to `v7.2.11`.
```

Comment:
Let's move this to bugs section and mention about the CVE. Is there any guidelines about not mentioning CVEs in release note?